config/application.rb

```diff
@@ -1,29 +1,36 @@
 require_relative "boot"
 
-require "active_job/railtie"
+require "rails"
```

Comment:
```suggestion
```
We don't need to require rails. We are already requiring the specific frameworks we need.

Reply:
:+1: